src/components/templates/PartyMap/PartyMap.scss

```diff
@@ -23,6 +23,10 @@
   .userlist-container {
     margin: 0 80px;
   }
+
+  @media (min-width: $large-laptop-width) {
+    width: calc(100% - #{$sidebar-width});
+  }
```

Comment:
This will only handle it for `PartyMap` template. We need to handle it for every template

Reply:
fixed